src/lib/entities.ts

```diff
@@ -13,9 +13,9 @@ export interface GithubRequestHeaders {
 
 export interface InitRepo {
   init: () => Promise<any>
-  createReadme: () => Promise<any>
-  createReactApp: () => Promise<any>
+  actions: any
```

Comment:
If we do as I suggested for the libraries, the actions could follow the same structure, i.e some kind of type `type Action = (dir, npx) => Promise<SOMETHING>`

we could then type this property :grimacing: